### PR TITLE
Update zig-wayland and zig-xkbcommon

### PR DESCRIPTION
**Pull request by fuzzypixelz**
I'm not sure all the generated protocol code is necessary for tinywl. But the compositor runs correctly.

### CODE REVIEW

File: tinywl/build.zig [hunk 1]
```diff
@@ -29,6 +29,17 @@ pub fn build(b: *Builder) void {
         .dependencies = &[_]Pkg{ wayland, xkbcommon, pixman },
     };
 
+    // These must be manually kept in sync with the versions wlroots supports
+    // until wlroots gives us the option request a specific version.
```
Comment (tiosgz) [inline tinywl/build.zig:33]: Typo inherited from river :)

```suggestion
    // until wlroots gives us the option to request a specific version.
```

**Comment by ifreund** [comment]
Please update the build.zig in the root of the repository as well so that `zig build test` passes.

**Comment by fuzzypixelz** [comment]
Should be good now.

**Review by ifreund — APPROVED** [review]
Thanks, I just squashed the commits and removed the xdg-ouput scanning cargo culted from river.

**Pull request merged**